src/Umbraco.Cms.Integrations.Crm.Hubspot/Umbraco.Cms.Integrations.Crm.Hubspot.csproj

```diff
@@ -42,6 +43,10 @@
 		</Content>
 	</ItemGroup>
 
+	<ItemGroup>
```

Comment:
Don't think you need this in the csproj... you should check, but I suspect it was added by Visual Studio unnecessarily.